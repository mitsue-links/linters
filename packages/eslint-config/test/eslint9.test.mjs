import path from 'path';
import {lintFile} from './helper.mjs';

describe('ESLint 9.0.0', () => {
    test('9.0.0_error.mjs', async () => {
        const filePath = '9.0.0_error.js';
        const data = await lintFile(path.join('fixtures', 'eslint', filePath));

        expect(data).toMatchSnapshot();
    });
});
