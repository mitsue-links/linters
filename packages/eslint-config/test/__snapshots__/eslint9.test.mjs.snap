// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ESLint 9.0.0 9.0.0_error.mjs 1`] = `
[
  {
    "filePath": "9.0.0_error.js",
    "messages": [
      {
        "line": 5,
        "ruleId": "no-useless-assignment",
        "severity": 2,
      },
    ],
  },
]
`;
